TELCODOCS-1893 SriovOperatorConfig will no longer be created by the Operator in 4.16 and onwards (#77080)
openshift/openshift-docs: modules/nw-sriov-configuring-operator.adoc

[id="nw-sriov-configuring-operator_{context}"]
= Configuring the SR-IOV Network Operator

> . Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components.

Reviewer: Unnumbered bullet when there is only one step. Also a colon at the end. Also is "related" critical, or could that be removed?

```suggestion
* Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator components:
```

> .  Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components .

Reviewer 2: ```suggestion
.  Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components.
```

.. Create a file named `sriovOperatorConfig.yaml` using the following YAML:
+
[source,yaml]
----
apiVersion: sriovnetwork.openshift.io/v1
kind: SriovOperatorConfig
metadata:
  name: default 
  namespace: openshift-sriov-network-operator 
spec:
  disableDrain: false
  enableInjector: true
  enableOperatorWebhook: true

>   logLevel: 2

Reviewer 2: @SchSeba are these spec parameters mandatory to be set? Do any of them override default parameter values?

Author: In this PR https://77149--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-reference-cluster-configuration-for-vdu.html#ztp-sno-du-configuring-sriov_sno-configure-for-vdu the YAML profile applied for SNO is slightly different enableOperatorWebhook: false and logLevel: 0

Reviewer 3: @cgoncalves @kquinn1204 This is the recommended configuration for multi node openshift.
for SNO the disableDrain MUST be true, and for ZTP they want to reduce CPU footprint so they also disable the webhooks 

we also have that if someone try to deploy the CR from the web console https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriov-network-operator.clusterserviceversion.yaml#L117

Author: @SchSeba thanks maybe not a biggie why the difference in LogLevel for ZTP where it is 0

----
+
[NOTE]
====

> The only valid name for the `sriovOperatorConfig` is `default` and it must be on the namespace where the operator is deployed. 

Reviewer: ```suggestion
The only valid name for the `SriovOperatorConfig` resource is `default` and it must be in the namespace where the Operator is deployed. 
```

====

> .. Create the object by running the following command:

Reviewer: I suggested adding "resource" above, but "object" would be fine too. But whichever you go with, I'd use the same for both

.Procedure

> - To set the `disableDrain` field to `true` and the `configDaemonNodeSelector` to `node-role.kubernetes.io/master: ""` , enter the following command:

Reviewer: ```suggestion
- To set the `disableDrain` field to `true` and the `configDaemonNodeSelector` field to `node-role.kubernetes.io/master: ""` , enter the following command:
```
